addons/io_scene_gltf2/__init__.py

```diff
@@ -1168,6 +1168,24 @@ def set_debug_log(self):
             self.loglevel = logging.NOTSET
 
 
+def update_gltf_settings_node_ui(self, context):
+    import io_scene_gltf2.blender.com.gltf2_blender_ui as blender_ui
+    if bpy.context.preferences.addons['io_scene_gltf2'].preferences.settings_node_ui is True:
+        bpy.utils.register_class(blender_ui.NODE_PT_GLTF_PANEL)
+    else:
+        blender_ui.unregister()
+
+class GLTF_AddonPreferences(bpy.types.AddonPreferences):
+    bl_idname = __package__
+
+    settings_node_ui : bpy.props.BoolProperty(default= False, update=update_gltf_settings_node_ui)
+
+
+    def draw(self, context):
+        layout = self.layout
+        row = layout.row()
+        row.prop(self, "settings_node_ui", text="Settings node interface")
```

Comment:
Suggested naming, and tooltip if possible:

- text: "Shader Editor Panel"
- tooltip: "Displays glTF 2.0 panel in the Sidebar region of the Shader Editor (shortcut for glTF Settings node)."

Based on capitalization elsewhere in preferences, and terminology used in the Shader Editor documentation: https://docs.blender.org/manual/en/latest/editors/shader_editor.html#sidebar

Reply:
Done, but need to be rechecked, as this is now in add menu